airflow/utils/log/secrets_masker.py

```diff
@@ -35,6 +35,7 @@
 DEFAULT_SENSITIVE_FIELDS = frozenset(
     {
         'access_token',
+        'token',
```

Comment:
Probably add it at the end of the list so this remains alphabetical

Reply:
updated to alphabetical - thanks.